Lib/idlelib/config-extensions.def

```diff
@@ -32,67 +32,6 @@
 # See config-keys.def for notes on specifying keys and extend.txt for
 # information on creating IDLE extensions.
 
-[AutoComplete]
-enable=True
-popupwait=2000
```

Comment:
As I said and explained on the issue, I want to leave non-key options in config-extensions.  So the 4 non-trivial [Xtension] sections should stay.  For former extensions, I would like to drop the requirement for an enable option when the entry is read in the editor.  Then we can delete the enable items.  I believe that this will be back-compatible with existing custom enable settings, but must verify.    (Ditto for not requiring some binding section.)

As I said yesterday, I now think the [Xtension_cfgBindings] should go, to be replaced by new entries in keysets.  This will require some change in IdleConf to deal better with missing entries in custom keysets.   But this was already needed, and perhaps should be a separate preliminary patch.

I think putting non-configurable key bindings in configuration files was a bit of a mistake.  In any case, these binding should be hard coded in the appropriate place.  At minimum, 'x_keys = idleConf(...)' should be replaced by 'x_keys = (current-idleconf-return).